cmd/agent/dist/conf.d/agentprofiling.d/conf.yaml.example

```diff
@@ -0,0 +1,32 @@
+## This file is overwritten upon Agent upgrade.
+## To make modifications to the check configuration, copy this file
+## to `conf.yaml` and make your changes on that file.
+
+init_config: {}
+instances:
+  - 
+
+    ## @param memory_threshold - integer - required - default: "0"
```

Comment:
Given that this can take a string like "10MB", should it be `string` instead of `integer`?